backend-forms.md

```diff
@@ -1081,7 +1081,7 @@ Method | Description
 **addFields** | adds new fields to the outside area
 **addTabFields** | adds new fields to the tabbed area
 **addSecondaryTabFields** | adds new fields to the secondary tabbed area
-**removeField** | remove a field from the tabbed area
+**removeField** | remove a field from the right area
```

Comment:
It would be more accurate to say `remove a field from any/all areas`: https://github.com/octobercms/october/blob/master/modules/backend/widgets/Form.php#L601

Reply:
I modified it.